lib/player.rb

```diff
@@ -3,16 +3,24 @@ def initialize(display)
     @display = display
   end
 
-  def selection
+  def selection(board)
     loop do
-      @display.output("Please select your move by entering the number (1 - 9) corresponding to an empty space.  The spaces are numbered from the top left (1) to the bottom right (9) of the board.")
-
+      @display.output("Please select your move by entering the number (1 - 9, from top left to bottom right) corresponding to an empty space.")
       selection = input_to_integer(@display.input)
-
-      selection.between?(1, 9) ? (return selection) : @display.output("Invalid entry.")
+      if selection_valid(selection)
+        return selection if selection_available(selection, board)
```

Comment:
Instead of the nested ifs, could we check the conditionals in one line? For ex. `if selection_valid(selection) && selection_available(selection, board)`

Reply:
Nice!  Thank you.